iosApp/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -16,44 +21,53 @@ struct LoginView: View {
 
     private let animationDuration: Double = 0.7
 
+    let coordinator: LoginCoordinator
+
     var body: some View {
         ZStack {
-            Assets.background
-                .image
-                .resizable()
-                .aspectRatio(contentMode: .fill)
-                .ignoresSafeArea()
-                .frame(maxWidth: .infinity, maxHeight: .infinity)
-
-            Assets.backgroundBlur
-                .image
-                .resizable()
-                .aspectRatio(contentMode: .fill)
-                .ignoresSafeArea()
-                .frame(maxWidth: .infinity, maxHeight: .infinity)
-                .opacity(animating ? 1.0 : 0.001)
-
-            VStack(
-                alignment: .center,
-                spacing: 20.0
-            ) {
-                Assets.logoWhite.image
-
-                Spacer().frame(maxHeight: 70.0)
-
-                if animating {
-                    loginField
-                    createPasswordField()
-                    loginButton
+            GeometryReader { geometry in
+                Assets.background
+                    .image
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .ignoresSafeArea()
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+
+                Assets.backgroundBlur
+                    .image
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .ignoresSafeArea()
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+                    .opacity(animating ? 1.0 : 0.001)
```

Comment:
Can't we set it to `0.0` when not animating? 🤔

Reply:
Added in https://github.com/blyscuit/kmm-swiftUI-IC/pull/56/commits/8edbbc2a81db456ef46a1258d58b109b6a24d252